references.bib

```diff
@@ -0,0 +1,278 @@
+  author={Algfoor, Zeyad Abd and Sunar, Mohd Shahrizal and Kolivand, Hoshang},
+  journal={International Journal of Computer Games Technology},
+  volume={2015},
+  pages={7},
+  year={2015},
+  publisher={Hindawi Publishing Corp.}
+}
+%used
+@inproceedings{kallmann2010navigation,
+  title={Navigation queries from triangular meshes},
+  author={Kallmann, Marcelo},
+  booktitle={International Conference on Motion in Games},
+  pages={230--241},
+  year={2010},
+  organization={Springer}
+}
+%used
+@article{korf1990real,
+  title={Real-time heuristic search},
+  author={Korf, Richard E},
+  journal={Artificial intelligence},
+  volume={42},
+  number={2-3},
+  pages={189--211},
+  year={1990},
+  publisher={Elsevier}
+}
+%used
+@article{sturtevant2007memory,
+  title={Memory-Efficient Abstractions for Pathfinding.},
+  author={Sturtevant, Nathan R},
+  journal={AIIDE},
+  volume={684},
+  pages={31--36},
+  year={2007}
+}
+%used
+@article{lawrence2013database,
+  title={Database-driven real-time heuristic search in video-game pathfinding},
+  author={Lawrence, Ramon and Bulitko, Vadim},
+  journal={IEEE Transactions on Computational Intelligence and AI in Games},
+  volume={5},
+  number={3},
+  pages={227--241},
+  year={2013},
+  publisher={IEEE}
+}
+%used
+@inproceedings{harabor2010breaking,
+  title={Breaking Path Symmetries on 4-Connected Grid Maps.},
+  author={Harabor, Daniel and Botea, Adi and others},
+  booktitle={AIIDE},
+  year={2010}
+}
+%used
+@article{lester2005pathfinding,
+  title={A* pathfinding for beginners},
+  author={Lester, Patrick},
+  journal={online]. GameDev WebSite. http://www. gamedev. net/reference/articles/article2003. asp (Acesso em 08/02/2009)},
+  year={2005}
+}
+%used
+@inproceedings{leigh2007using,
+  title={Using a genetic algorithm to explore A*-like pathfinding algorithms},
+  author={Leigh, Ryan and Louis, Sushil J and Miles, Chris},
+  booktitle={Computational Intelligence and Games, 2007. CIG 2007. IEEE Symposium on},
+  pages={72--79},
+  year={2007},
+  organization={IEEE}
+}
+%used
+@inproceedings{harabor2011online,
+  title={Online Graph Pruning for Pathfinding On Grid Maps.},
+  author={Harabor, Daniel Damir and Grastien, Alban and others},
+  booktitle={AAAI},
+  year={2011}
+}
+%used
+@book{millington2016artificial,
+  title={Artificial intelligence for games},
+  author={Millington, Ian and Funge, John},
+  year={2016},
+  publisher={CRC Press}
+}
+%used
+@article{wang2012game,
+  title={Game ai: Simulating car racing game by applying pathfinding algorithms},
+  author={Wang, Jung-Ying and Lin, Yong-Bin},
+  journal={International Journal of Machine Learning and Computing},
+  volume={2},
+  number={1},
+  pages={13},
+  year={2012},
+  publisher={IACSIT Press}
+}
+%used
+@misc{harabor2012fast,
+  title={Fast pathfinding via symmetry breaking},
+  author={Harabor, Daniel},
+  journal={2013-11-20]. http://aigamedev. com/open/tutorial/symmetry-in-pathfinding},
```

Comment:
BibTeX ignores this -- in a `@misc`, you need to put the URL in a `howpublished` field. In general, make sure you check the compiled BibTeX output to ensure it's including all the required information.